[2025-02-12T22:30:38.850+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T22:30:38.864+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T02:02:00+00:00 [queued]>
[2025-02-12T22:30:38.872+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T02:02:00+00:00 [queued]>
[2025-02-12T22:30:38.872+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T22:30:38.889+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-04 02:02:00+00:00
[2025-02-12T22:30:38.896+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=14036) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T22:30:38.897+0000] {standard_task_runner.py:72} INFO - Started process 14046 to run task
[2025-02-12T22:30:38.896+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-04T02:02:00+00:00', '--job-id', '4992', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmp62rfamhi']
[2025-02-12T22:30:38.898+0000] {standard_task_runner.py:105} INFO - Job 4992: Subtask fetch_and_store_crypto_data
[2025-02-12T22:30:38.937+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T02:02:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T22:30:39.023+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-04T02:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-04T02:02:00+00:00'
[2025-02-12T22:30:39.024+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T22:30:41.565+0000] {logging_mixin.py:190} INFO - Données récupérées : {'Response': 'Error', 'Message': 'You are over your rate limit please upgrade your account!', 'HasWarning': False, 'Type': 99, 'RateLimit': {'calls_made': {'second': 1, 'minute': 80, 'hour': 3579, 'day': 4473, 'month': 4473, 'total_calls': 4473}, 'max_calls': {'second': 20, 'minute': 300, 'hour': 3000, 'day': 7500, 'month': 11000}}, 'Data': {}}
[2025-02-12T22:30:41.568+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans PostgreSQL : string indices must be integers, not 'str'
[2025-02-12T22:30:41.583+0000] {logging_mixin.py:190} INFO - Données uploadées dans MinIO.
[2025-02-12T22:30:41.584+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T22:30:41.590+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T22:30:41.591+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-04T02:02:00+00:00, execution_date=20231004T020200, start_date=20250212T223038, end_date=20250212T223041
[2025-02-12T22:30:41.657+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T22:30:41.683+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T22:30:41.685+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
